person.rb

```diff
@@ -10,6 +10,7 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
```

Comment:
- [ ] Kindly consider adding a method add_rental in this class after initialize method. In that method you will be getting the parameters of (book,date) and inside that method, you will be initializing and returning a rental object where you will be also giving the self person class to make the association right. example: Rental.new(date, self, book)